paasta_tools/docker_wrapper.py

```diff
@@ -296,11 +297,15 @@ def main(argv=None):
         else:
             argv = add_argument(argv, '--mac-address={}'.format(mac_address))
 
-            prepare_new_container(
-                DEFAULT_SOA_DIR,
-                DEFAULT_SYNAPSE_SERVICE_DIR,
-                env_args['PAASTA_SERVICE'],
-                env_args['PAASTA_INSTANCE'],
-                mac_address)
+            try:
+                with firewall_flock():
+                    prepare_new_container(
+                        DEFAULT_SOA_DIR,
+                        DEFAULT_SYNAPSE_SERVICE_DIR,
+                        env_args['PAASTA_SERVICE'],
+                        env_args['PAASTA_INSTANCE'],
+                        mac_address)
+            except Exception as e:
+                print('Unable to add firewall rules: {}'.format(e), file=sys.stderr)
```

Comment:
I'm sort of worried about this breaking and us never noticing. I think it's right to not block container startup (could lead to a nasty outage), but I wonder if there's any way we can know when this is happening?

Failing open like this isn't the kind of failure anyone would ever really notice, I think.